src/webextension/background/accounts/configs.json

```diff
@@ -0,0 +1,32 @@
+{
+  "production": {
+    "client_id": "1b024772203a0849",
+    "redirect_uri": "https://2aa95473a5115d5f3deb36bb6875cf76f05e4c4d.extensions.allizom.org/",
+    "authorization_endpoint": "https://oauth.accounts.firefox.com/v1/authorization",
+    "token_endpoint": "https://oauth.accounts.firefox.com/v1/token",
+    "userinfo_endpoint": "https://profile.accounts.firefox.com/v1/profile",
```

Comment:
FWIW it should now be possible to automagically discover these URLs from https://accounts.firefox.com/.well-known/openid-configuration (or similar in dev) which has recently been updated with proper cross-origin support.

Reply:
@rfk awesome!

Is it a showstopper if dynamic discover lands in the next iteration?  This work is really big, and I'm hoping discovery can wait until this settles ...

Reply:
No, not a showstopper at all, just making sure you're aware that CORS bug was fixed :-)

Reply:
tracking this via #380